cookiecutter/generate.py

```diff
@@ -17,6 +17,7 @@
 import shutil
 
 from jinja2 import FileSystemLoader
+from jinja2._compat import string_types
```

Comment:
We're using `python-future`, so you can use [`from past.builtins import basestring`](http://python-future.org/compatible_idioms.html#basestring) instead of using the [private](https://docs.python.org/3.5/tutorial/classes.html#private-variables) jinja2 module.